cli/pbjs.js

```diff
@@ -148,6 +149,7 @@ exports.main = function main(args, callback) {
                 "  --force-message  Enforces the use of message instances instead of plain objects.",
                 "",
                 "  --null-defaults  Default value for optional fields is null instead of zero value.",
+                "  --pb3-optional   Make type declarations respect optional fields for PB3.",
```

Comment:
May be just "--disable-defaults" and thats all? There is no data in grpc package on default value, so, why we should have some magic default or unexpected null, it is undefined and thats all. So, just disable defaults and it'll be ok)